scripts/scenarios/cloud6/qa-scenario-6a-sbd-kvm-docker.yaml

```diff
@@ -0,0 +1,231 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    stonith:
+      mode: sbd
+      sbd:
+        nodes:
+          d52-54-77-77-77-01.v6.cloud.suse.de:
+            devices:
+            - "/dev/sda"
+          d52-54-77-77-77-02.v6.cloud.suse.de:
+            devices:
+            - "/dev/sda"
+          d52-54-77-77-77-03.v6.cloud.suse.de:
+            devices:
+            - "/dev/sda"
+      per_node:
+        nodes:
+          d52-54-77-77-77-01.v6.cloud.suse.de:
+            params: ''
+          d52-54-77-77-77-02.v6.cloud.suse.de:
+            params: ''
+          d52-54-77-77-77-03.v6.cloud.suse.de:
+            params: ''
+  deployment:
+    elements:
+      pacemaker-cluster-member:
+      - "@@controller1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+      hawk-server:
+      - "@@controller1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+# WARNING: no proposals exist for nfs_client barclamp
+# WARNING: no proposals exist for suse_manager_client barclamp
+- barclamp: database
+  attributes:
+    ha:
+      storage:
+        shared:
+          device: <nfs-server-ip>:/exportpg
+          fstype: nfs
+  deployment:
+    elements:
+      database-server:
+      - cluster:services
+- barclamp: rabbitmq
+  attributes:
+    password: uz8FaasWz9st
+    ha:
+      storage:
+        shared:
+          device: <nfs-server-ip>:/exportrabbit
```

Comment:
I'm not sure about this one. But maybe it is better than using some specific address that is QA specific.